apps/omg_watcher/test/exit_processor/core_test.exs

```diff
@@ -90,14 +92,14 @@ defmodule OMG.Watcher.ExitProcessor.CoreTest do
       transactions
       |> Enum.map(&Transaction.encode/1)
 
-    [tx1_sigs, tx2_sigs] =
+    [tx1_signs, tx2_signs] =
```

Comment:
can we undo this? we use `sigs` everywhere I think, `signs` is an English word, so it's actually more confusing

Reply:
done